lib/index.js

```diff
@@ -17,6 +16,13 @@ const util = require('util')
  */
 const rm      = util.promisify(require('rimraf'))
 
+
+/**
+ * Helpers
+ */
+const absolute = (s) => {return path.resolve(s) === s}
```

Comment:
I looked at the code for `absolute`. It's a polyfill for [`path.isAbsolute`](https://nodejs.org/api/path.html#path_path_isabsolute_path), which was added in node 0.12.

Given that we don't support versions of node older than that anymore, you can shorten this line further to `const absolute = path.isAbsolute.bind(path)` or just get rid of it entirely and replace the calls further down with calls to `path.isAbsolute`.